lib/data.js

```diff
@@ -0,0 +1,22 @@
+const data = {
```

Comment:
get fuctions live in `lib/` and fetch from the mock data sitting in `public/` currently. Had a blocker where plopping the JSON file into `pages/api/` started appending `.index.tsx` to the endpoint.